ui/src/common/utils.js

```diff
@@ -168,3 +169,33 @@ export function pickEvenlyDistributedElements(array, numberOfElements) {
   }
   return someElements;
 }
+
+// adapted from https://developer.mozilla.org/en-US/docs/Web/API/Window/navigator
+// use a library (npmjs.com/package/bowser) if more info from userAgent is needed (like OS, version etc.)
+export function getBrowserName(userAgent) {
```

Comment:
can we just return an enum instead of string?

Reply:
maybe if we used `TypeScript` ;)

Reply:
yes let's move them to ``const`` or something